packages/playground/src/lib/command/navigation-command.css

```
@@ -0,0 +1,126 @@
+.navigation-command-popup[data-open] {
+  animation: none !important;
+}
+
+.navigation-command-popup[data-closed] {
+  animation: navigation-command-popup-hold 260ms linear both !important;
+}
+
+.navigation-command-surface {
+  --navigation-command-enter-delay: 0ms;
+  --navigation-command-exit-delay: 0ms;
+  transform-origin: center;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)